roles/entitlement_deploy/files/apply_template.py

```diff
@@ -1,19 +1,33 @@
-#! /usr/bin/python3
+#!/usr/bin/env python3
 
-import fileinput
 import base64
 import sys
 
-template = sys.argv[1]
-keyname = sys.argv[2]
-filename = sys.argv[3]
-print(f"Replacing '{keyname}' with the content of '{filename}'", file=sys.stderr)
 
-with open(filename, "rb") as f:
-    file_b64 = base64.b64encode(f.read()).decode()
+def main():
+    keyname = sys.argv[1]
+    value = sys.argv[2]
 
-found = False
-for line in fileinput.input(files=[template]):
-    if keyname in line: found = True
-    print(line.replace(keyname, file_b64), end="")
-exit(0 if found else 1)
+    if value.startswith("@"):
```

Comment:
Added support for either reading the contents from a file, or simply using the argument passed to it. Because I just want to change a VAR name to "worker" or "master"

Reply:
could you add a line of comment explaining what the script does / how the parameters are used?

I understood after reading the task calling the script, the update makes sense

I'm wondering if we should use Ansible `template` task instead, or not